style.css

```diff
@@ -0,0 +1,674 @@
+/* colors: #272a31-dark, #ec5242 - orange, #d3d3d3 - lightgrey */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+  font-size: 1.2rem;
+  color: #272a31;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+ul {
+  list-style-type: none;
+}
+
+h2 {
+  font-size: 2rem;
+}
+
+.centered {
+  text-align: center;
+}
+
+.header {
+  position: relative;
+}
+
+.desktop-navigation {
+  display: none;
+}
+
+.mobile-nav {
+  display: flex;
+  padding: 15px;
+  font-size: 2.5rem;
+  position: fixed;
+  width: 100%;
+  background: url('./images/background.png');
+}
+
+.mobile-menu {
+  display: none;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background: #272a31f1;
+  padding: 15px;
+}
+
+.mobile-menu li {
+  /* margin: 15px 0; */
+  border-bottom: 1px solid #ec5242;
+  padding: 0;
+}
+
+.close {
+  color: #ec5242;
+  font-size: 2rem;
+  margin-bottom: 15px;
+}
+
+.mobile-menu a {
+  color: #c9c7c7;
+  font-size: 2rem;
+  padding: 15px 0;
+
+  /* margin: 15px; */
```

Comment:
- [ ] Remove the commented code. Removing commented code make codebase more cleaner.